geth/jail/jail_test.go

```diff
@@ -119,6 +125,36 @@ func (s *JailTestSuite) TestFunctionCall() {
 	require.Equal(expectedResponse, response)
 }
 
+func (s *JailTestSuite) TestJailRPCAsyncSend() {
```

Comment:
Shouldn't we make a simple test which also checks if a callback is called? I am not sure what we're testing here as `web3.eth.sendTransaction(data, function(){})` would be called without these changes anyway, just in sync way I guess.

Reply:
It's checked everywhere in a lot of tests, this is why I decided not to include another test.
I'll write a comment what this test is about.